python/lgbserver/lgbserver/model.py

```diff
@@ -20,7 +20,7 @@
 
 from kserve import Model
 from kserve.errors import InferenceError, ModelMissingError
-from kserve.storage import Storage
+from storage import Storage
```

Comment:
In the kserve/storage directory, we can do something like this for backward compatibility
```
__init__.py
    try:
       from storage import *
    except ImportError as e:
       Print("Please install storage package")
       raise from e
       
```

Reply:
That's a good idea, but I was not able to make it work, can you please take a look here and see what I might be missing: 
https://github.com/spolti/kserve/commit/c54fcd37464a1a32f08f596b9081c93c175f523b

Reply:
@sivanantha321 I guess it worked now, can you please check the last commit?

Reply:
Just adding the storage module as optional should be enough not to break backward compatibility.